packages/components/bucket-policy/create-bucket-policy.tsx

```diff
@@ -168,20 +184,17 @@ const CreateBucketPolicy: React.FC<CreateBucketPolicyProps> = ({
   return (
     <>
       <Modal {...modalProps} />
-      <div className="co-create-operand__header">
-        <Title
-          size="2xl"
-          headingLevel="h1"
-          className="co-create-operand__header-text"
-        >
-          {t('Create new Bucket')}
-        </Title>
-        <p className="help-block">
+      <PageHeading
+        breadcrumbs={breadcrumbs}
+        title={t('Create new Bucket')}
+        className="breadcrumbs-header"
+      >
+        <p>
           {t(
             'A set of policies that would apply to all buckets (OBCs) created with the specific bucket policy. These policies include namespace and caching'
```

Comment:
Just a doubt:
  Will the user know what is OBC means? are we documenting this somewhere?

Reply:
it was there in UX as OBCs. hopefully, they are aware of it

Reply:
ok